### ISSUE TITLE
Build failure: shiboken2-5.15.11

### ISSUE THREAD

**Issue by Brojan96** [NixOS/nixpkgs#326881]
### Steps To Reproduce

Sorry this is my first issue on github. Hope I am doing this correctly.

1. `sudo nixos-rebuild switch --upgrade` with the following nixos config:
```
environment.systemPackages = with pkgs; [
    adwaita-icon-theme
    android-tools
    authenticator
    baobab
    #bash-completion
    bitwarden
    bottles
    btop
    celluloid
    chromium
    cinnamon.warpinator
    curl
    #dconf2nix
    dconf-editor
    discord
    element-desktop
    firefox
    flameshot
    freecad
    git
    github-desktop
    gnome-tweaks
    gnomeExtensions.alphabetical-app-grid
    gnomeExtensions.tiling-assistant
    heroic
    jellyfin-media-player
    lshw
    lsof
    lutris
    mediawriter
    mkvtoolnix
    mullvad-vpn
    mumble
    neofetch
    notepad-next
    nix-bash-completions
    onlyoffice-bin
    picard
    protonup
    protonup-qt
    qbittorrent
    #skypeforlinux
    signal-desktop
    tagger
    telegram-desktop
    thunderbird
    tldr
    toybox
    transmission_4
    unzip
    usbimager
    virt-manager
    vscodium
    wget
    whatsapp-for-linux
    xz
    #zsh
  ];
```

### Build log

```
unpacking channels...
building Nix...
building the system configuration...
error:
       … while calling the 'head' builtin

         at /nix/var/nix/profiles/per-user/root/channels/nixos/lib/attrsets.nix:1575:11:

         1574|         || pred here (elemAt values 1) (head values) then
         1575|           head values
             |           ^
         1576|         else

       … while evaluating the attribute 'value'

         at /nix/var/nix/profiles/per-user/root/channels/nixos/lib/modules.nix:821:9:

          820|     in warnDeprecation opt //
          821|       { value = addErrorContext "while evaluating the option `${showOption loc}':" value;
             |         ^
          822|         inherit (res.defsFinal') highestPrio;

       (stack trace truncated; use '--show-trace' to show the full trace)

       error: Package ‘shiboken2-5.15.11’ in /nix/store/98mgnj8fy0vx0l7yq1lzls93d5h6zhmv-nixos/nixos/pkgs/development/python-modules/shiboken2/default.nix:54 is marked as broken, refusing to evaluate.

       a) To temporarily allow broken packages, you can use an environment variable
          for a single invocation of the nix tools.

            $ export NIXPKGS_ALLOW_BROKEN=1

          Note: When using `nix shell`, `nix build`, `nix develop`, etc with a flake,
                then pass `--impure` in order to allow use of environment variables.

       b) For `nixos-rebuild` you can set
         { nixpkgs.config.allowBroken = true; }
       in configuration.nix to override this.

       c) For `nix-env`, `nix-build`, `nix-shell` or any other Nix command you can add
         { allowBroken = true; }
       to ~/.config/nixpkgs/config.nix.
```

### Additional context

Using nixos unstable.

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.9.7-zen1, NixOS, 24.11 (Vicuna), 24.11pre647193.9f4128e00b0a`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.4`
 - channels(root): `"nixos"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

---

**Comment by justgeig** [comment]
experiencing this issue as well with trying to switch from 24.05 to unstable

**Comment by jerith666** [comment]
This looks to be fixed in #326595.

**Comment by squalus** [comment]
Duplicate of https://github.com/NixOS/nixpkgs/issues/325832
